app/components/templates/Main/Main.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import Nav from '../../organisms/Nav/Nav';
+import Link from '../../../components/atoms/Link/Link.js'
+
+export default class Main extends React.Component {
+  constructor(props) {
+      super(props);
+  }
+
+  render() {
+    return(
+      <div className="app">
+        <Nav />
+        
+        <main>
```

Comment:
That's fine. We can take it out

Yes, I think we should cut it down too. I wanted to throw it in there to test the activeState with the Router, but that can be done for later probably.